pointmatcher/IO.cpp

```diff
@@ -1524,6 +1524,10 @@ typename PointMatcherIO<T>::DataPoints PointMatcherIO<T>::loadPLY(std::istream&
 			const int row = vertex->properties[propID].pmRowID;
 			const PMPropTypes type = vertex->properties[propID].pmType;
 			
+			if (vertex->properties[propID].name == "red" || vertex->properties[propID].name == "green" || vertex->properties[propID].name == "blue" || vertex->properties[propID].name == "alpha") {
```

Comment:
The unit test was failing because the savePLY had the same problem as the loadPLY.
Analyzing the IO class, it seems that colors are assumed to be in range [0,1] in float format.
But ply files store each channel as an unsigned char in range [0,255].
The modifications i introduced take this ply format detail in consideration.